vignettes/cookbook_metrics.Rmd

```diff
@@ -0,0 +1,219 @@
+
+We define the "Technology Market Share Ratio" as:
+
+$$\dfrac{s_i(t)}{s_i(t_0)}$$
+This method is used to set targets for "decreasing" (i.e. brown) technologies. 
+
+#### 2. Market-share by sector
+To calculate the market-share by sector, we use the initial production of both the portfolio and scenario at the sector-level instead. 
+$$p_{i}^{smsp}(t) = p_{i}(t_0) +P(t_0) * \left( \dfrac{s_i(t)-s_i(t_0)}{S(t_0)}\right)$$
+where:
+
+* $P_i(t_0)$ is the portfolio's total production in the sector at $t_0$, and
+* $S(t_0)$ is the scenario total production at $t_0$.
+
+We define the "Sector Market Share Percentage" as:
+
+$$\dfrac{s_i(t)-s_i(t_0)}{S(t_0)}$$
+This method is used to calculate targets for "increasing" (i.e. green) technologies. 
+
+### How to calculate market-share targets for a given scenario
+
+To calculate market-share targets, you need to use the package `{pacta.loanbook}` and a number of datasets, including a "matched" dataset (loanbook + asset-level data) that you can get by following the [Matching Process](cookbook_running_the_analysis.html#matching-process) section of the [Running the Analysis](cookbook_running_the_analysis.html) chapter of the cookbook. The datasets used here are included in the `{pacta.loanbook}` package; they are fake but show how you should structure your own data.
+
+* Load the package.
+
+```{r, message = FALSE}
+library(pacta.loanbook)
+```
+
+* Match the loanbook to asset level data.
+
+```{r, R.options = list(width = 400)}
+loanbook <- loanbook_demo
+abcd <- abcd_demo
+
+matched <- match_name(loanbook = loanbook, abcd = abcd) %>%
+  # WARNING: Remember to validate the output of match_name() before prioritize()
+  prioritize()
+
+matched
+```
+
+* Calculate market-share targets for production at the portfolio level.
+
+```{r, R.options = list(width = 400)}
+# portfolio level targets
+scenario <- scenario_demo_2020
+regions <- region_isos_demo
+
+
+matched %>%
+  target_market_share(
+    abcd = abcd,
+    scenario = scenario,
+    region_isos = regions
+  )
+```
+
+* Calculate market-share targets for production at the company level.
+
+```{r, R.options = list(width = 400)}
+matched %>%
+  target_market_share(
+    abcd = abcd,
+    scenario = scenario,
+    region_isos = regions,
+    by_company = TRUE,
+    weight_production = FALSE
+  )
+```
+
+## Sectoral Decarbonization Approach
+
+The Sectoral Decarbonization Approach (SDA) is a method for setting corporate CO^2^ emissions intensity reduction targets in line with climate science. This method was developed by the Science-Based Targets Initiative ([SBTI](https://sciencebasedtargets.org/)), an international initiative on science-based target setting for companies initiated by [CDP](https://www.cdp.net/en), the [United Nations Global Compact](https://unglobalcompact.org/), the World Resources Institute ([WRI](https://www.wri.org/)), and the Worldwide Fund for Nature ([WWF](https://www.worldwildlife.org/)). 
```

Comment:
Throughout this document, we write CO2, with the 2 in superscript, but it needs to be subscript. Can be a follow up task though

Reply:
ack! I'll fix it

Reply:
fixed